prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -409,37 +408,18 @@ class WorkflowEngine(
         val errors: MutableList<ResultDetail> = mutableListOf()
         val warnings: MutableList<ResultDetail> = mutableListOf()
 
-        val reportFile = db.fetchReportFile(messageEvent.reportId)
-
         db.transact { txn ->
             val task = db.fetchAndLockTask(messageEvent.reportId, txn)
 
-            val blobContent = blob.downloadBlob(reportFile.bodyUrl)
+            val blobContent = blob.downloadBlob(task.bodyUrl)
```

Comment:
might be able to change this back.

Reply:
Task is probably a better idea, since we really are processing a 'task' here, which has a report attached to it.